lib/new_relixir/statman.ex

```diff
@@ -0,0 +1,186 @@
+defmodule NewRelixir.Statman do
+  def poll do
+    [end_time, start_time, metrics, errors] = NewRelixir.Collector.poll
+    transform_aggregated_metrics(metrics, errors, {start_time, end_time})
+  end
+
+  def transform_aggregated_metrics(metrics, errors, time) do
+    ms = metrics
+      |> Map.to_list
+      |> Enum.reduce([], fn(m, acc) ->
+        acc ++ transform_metric(m)
+      end)
+      |> Enum.filter(&(&1 != []))
+
+    errs = errors |> Map.to_list |> Enum.map(fn(metric) -> transform_error_counter(metric) end)
+
+    {[webtransaction_total(ms), db_total(ms) | errors_total(errs) ++ ms], errs, time}
+  end
+
+  def transform_error_counter({{scope, type, message}, count}) do
+    error = [
+      :os.system_time(:micro_seconds) / 1_000_000,
+      scope2bin(scope),
+      to_bin(message),
+      to_bin(type),
+      [%{
+        parameter_groups: %{},
+        stack_trace: [],
+        request_params: %{},
+        request_uri: scope}]
+    ]
+    List.duplicate(error, count) |> List.flatten
+  end
+
+  defp transform_metric(metric) do
```

Comment:
This function doesn't seem to add much value, we should probably call `transform_histogram` straight.

Reply:
Done in def81565faac1b97b15e0d9e48a4ad8892c40848